# Mean Median Mode คือการหาค่ามัธยฐานเฉลี่ย


In [2]:
import numpy as np

speed = [99,86,87,88,111,86,103,87,94,78,77,85,86] # ข้อมูลความเร็วในแต่ละรอบ

x = np.mean(speed) # คำสั่งหาค่ามัธยฐานเฉลี่ยน

print(x)

89.76923076923077


## การทำงานของ .mean()

คือจะนำผลข้อมูลทั้งหมดมารวมกันแล้วหารด้วยจำนวนทั้งหมด

(99+86+87+88+111+86+103+87+94+78+77+85+86) / 13 = 89.77